x-pack/test/functional/apps/security/rbac_phase1.js

```diff
@@ -73,47 +75,32 @@ export default function ({ getService, getPageObjects }) {
 
     //   this is to acertain that all role assigned to the user can perform actions like creating a Visualization
     it('rbac all role can save a visualization', async function () {
-      const fromTime = '2015-09-19 06:31:44.000';
-      const toTime = '2015-09-23 18:31:44.000';
-      const vizName1 = 'Visualization VerticalBarChart';
-
       log.debug('navigateToApp visualize');
       await PageObjects.security.login('kibanauser', 'changeme');
       await PageObjects.common.navigateToUrl('visualize', 'new');
       log.debug('clickVerticalBarChart');
       await PageObjects.visualize.clickVerticalBarChart();
       await PageObjects.visualize.clickNewSearch();
-      log.debug('Set absolute time range from \"' + fromTime + '\" to \"' + toTime + '\"');
-      await PageObjects.header.setAbsoluteRange(fromTime, toTime);
```

Comment:
This setup isn't necessary for this test to make its assertions. I removed it to save some time.